Common.sln

```diff
@@ -17,6 +17,8 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Framework.Propert
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Framework.Internal.Test", "test\Microsoft.Framework.Internal.Test\Microsoft.Framework.Internal.Test.kproj", "{D81EAC23-B515-49FB-B04B-285A9C1C6583}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Framework.NullLogging.Internal", "src\Microsoft.Framework.NullLogging.Internal\Microsoft.Framework.NullLogging.Internal.kproj", "{4917B89F-2714-4912-B8BB-3C955A0CA691}"
```

Comment:
Need to change name to `Microsoft.Framework.NullLogger.Internal` and add another project `Microsoft.Framework.NullLoggerFactory.Internal`.